main.py

```diff
@@ -10,7 +10,9 @@
 import pandas as pd
 from request_generator import RequestGenerator
 from request import Request
-from PyQt5 import uic, QtWidgets"""to bedzie do GUI"""
+#from PyQt5 import uic, QtWidgets  '''to bedzie do GUI'''

```

Comment:
To tylko przypadkiem zostawiłam zakomentowane. U mnie wywalało błędy i nic z tym nie robiłam